pkg/machine/applehv/machine.go

```diff
@@ -18,28 +32,596 @@ func GetVirtualizationProvider() machine.VirtProvider {
 	return &Virtualization{
 		artifact:    machine.None,
 		compression: machine.Xz,
-		format:      machine.Qcow,
+		format:      machine.Raw,
 	}
 }
 
-const (
-	// Some of this will need to change when we are closer to having
-	// working code.
-	VolumeTypeVirtfs     = "virtfs"
-	MountType9p          = "9p"
-	dockerSock           = "/var/run/docker.sock"
-	dockerConnectTimeout = 5 * time.Second
-	apiUpTimeout         = 20 * time.Second
-)
+// VfkitHelper describes the use of vfkit: cmdline and endpoint
+type VfkitHelper struct {
+	Bootloader        string
+	Devices           []string
+	LogLevel          logrus.Level
+	PathToVfkitBinary string
+	Endpoint          string
+}
 
-type apiForwardingState int
+type MacMachine struct {
+	// ConfigPath is the fully qualified path to the configuration file
+	ConfigPath machine.VMFile
+	// HostUser contains info about host user
+	machine.HostUser
+	// ImageConfig describes the bootable image
+	machine.ImageConfig
+	// Mounts is the list of remote filesystems to mount
+	Mounts []machine.Mount
+	// Name of VM
+	Name string
+	/*
+		// NetworkVSock is for the user networking
+		NetworkHVSock machine.HVSockRegistryEntry
```

Comment:
Why is this commented? Can it be removed?

Reply:
no, holding onto it as part of the networking decisions that need to be made.  you will see that as a theme in the code.